src/Nordea/Components/NumberInput.elm

```diff
@@ -152,14 +165,25 @@ getStyles config =
 
             else
                 Colors.mediumGray
+
+        sizeSpecificStyling =
+            case config.size of
+                Small ->
+                    [ fontSize (rem 0.75)
+                    , height (rem 1.5)
+                    , padding2 (rem 0.25) (rem 0.5)
```

Comment:
Is it correct that it has more padding than the standard size?☺️

Reply:
I think this is correct. Has less padding on the left and right `0.5` then the standard `0.75`. The padding on top/bottom doesn't make much difference here. I had the values from design so have included it. We can also set it `rem 0`

Reply:
I see, if Lines design has 0.25 then I think we should stick with that, was just curious😁

Reply:
Good comment though. So, then lets keep the top/bottom padding ☺️